python/pip_install/pip_repository_requirements_bzlmod.bzl.tmpl

```diff
@@ -22,3 +22,18 @@ def data_requirement(name):
 
 def dist_info_requirement(name):
     return "%%MACRO_TMPL%%".format(_clean_name(name), "dist_info")
+
+def entry_point(pkg, script = None):
+    """entry_point returns the target of the canonical label of the package entrypoints.
+
+    To use it one has to explicitly use the repo for the entrypoint, e.g.:
+    ```
+    use_repo(
+        "pypi",
+        "pypi_yamllint",
+    )
+    ```
+    """
+    if not script:
+        script = pkg
+    return "@@%%NAME%%_{}//:rules_python_wheel_entry_point_{}".format(_clean_name(pkg), script)
```

Comment:
You should be able to use `@%%FRIENDLY_NAME%%` instead, where `FRIENDLY_NAME` is what the user passed in as `name`. You can get it by making the repository rule a macro that sets both the actual `name` attr and a `friendly_name` `attr.string()` to the value passed in as `name`.